src/components/grouped-character/grounded-character.test.tsx

```diff
@@ -0,0 +1,604 @@
+import React, { useState } from "react";
+import { render, screen } from "@testing-library/react";
+
+import userEvent from "@testing-library/user-event";
+import GroupedCharacter from "./grouped-character.component";
+import { testStyledSystemMargin } from "../../__spec_helper__/__internal__/test-utils";
+import FormFieldStyle from "../../__internal__/form-field/form-field.style";
+import Logger from "../../__internal__/utils/logger";
+
+jest.mock("../../__internal__/utils/logger");
+
+jest.useFakeTimers();
```

Comment:
**suggestion(non-blocking)**: what do you think about amending the fake timer setup like so, to ensure timer code from one test doesn’t affect the next?   

```suggestion
beforeEach(() => {
  jest.useFakeTimers();
})

afterEac(() => {
  jest.runOnlyPendingTimers();
  jest.useRealTimers();
})
```